tests/test_modeling_bert.py

```diff
@@ -152,7 +152,7 @@ def prepare_config_and_inputs_for_decoder(self):
             encoder_attention_mask,
         )
 
-    def create_and_check_bert_model(
+    def create_and_check_model(
```

Comment:
shorten name -> no need to add `_bert` here because all tests are only relevant for BERT

Reply:
Completely agree with this renaming.

Reply:
Nice!